config.json

```diff
@@ -612,6 +612,18 @@
       "unlocked_by": "rotational-cipher",
       "uuid": "d36ce010-210f-4e9a-9d6c-cb933e0a59af"
     },
+    {
+      "core": false,
+      "difficulty": 5,
```

Comment:
As described in our [policies doc](https://github.com/exercism/java/blob/master/POLICIES.md#add-hint-for-the-first-exercises-without-starter-implementation) all exercises with difficulty 5 should have a hint added to them since they are the first exercises without starter implementation 🙂 

However, this exercise is fairly complicated so I think it should maybe have a difficulty higher than 5. What do you think @jssander, @exercism/java? 🙂

Reply:
What number do you think it should have?

Reply:
Maybe 7 or 8? What do you think @sjwarner-bp? 🙂

Reply:
I think this could be a quite difficult task... Maybe a 7? 🙂

Reply:
I think an 8 might be a little high, so I'd go with a 7

Reply:
Sounds like 7 would be a good idea then! 🙂